docs/docs/consumer-development/changeover-procedure.md

```diff
@@ -0,0 +1,231 @@
+
+* `chain_id` must be equal to the sovereign chain id
+* `initial_height` field has additional rules to abide by:
+
+:::caution
+```
+{
+...
+    "initial_height" : {
+        // must correspond to current revision number of sovereign chain
+        // e.g. stride-1 => "revision_number": 1
+        "revision_number": 1,
+
+        // must correspond to a height that is at least 1 block after the upgrade
+        // that will add the `consumer` module to the sovereign chain
+        // e.g. "upgrade_height": 100 => "revision_height": 101
+        "revision_height": 1,
+    },
+...
+}
+```
+RevisionNumber: 0, RevisionHeight: 111
+:::
+
+* `genesis_hash` can be safely ignored because the chain is already running. A hash of the sovereign chain's initial genesis may be used
+
+* `binary_hash` may not be available ahead of time. All chains performing the changeover go through rigorous testing - if bugs are caught and fixed the hash listed in the proposal may not be the most recent one.
+
+* `spawn_time` listed in the proposal MUST be before the `upgrade_height` listed in the the upgrade proposal on the sovereign chain.
+:::caution
+`spawn_time` must occur before the `upgrade_height` on the sovereign chain is reached becasue the `provider` chain must generate the `ConsumerGenesis` that contains the **validator set** that will be used after the changeover.
+:::
+
+* `unbonding_period` must correspond to the value used on the sovereign chain. Otherwise, the clients used for the ccv protocol may be incorrectly initialized.
+
+* `distribution_transmission_channel` **should be set**.
+
+:::note
+Populating `distribution_transmission_channel` will enable the sovereign chain to re-use one of the existing channels to the provider for consumer chain rewards distribution. This will preserve the `ibc denom` that may already be in use.
+
+If the parameter is not set, a new channel will be created.
+:::
+
+* `ccv_timeout_period` has no important notes
+
+* `transfer_timeout_period` has no important notes
+
+* `consumer_redistribution_fraction` has no important notes
+
+* `blocks_per_distribution_transmission` has no important notes
+
+* `historical_entries` has no important notes
+
+
+### 2. upgrade proposal on sovereign chain
+
+The sovereign chain creates an upgrade proposal to include the `interchain-security/x/ccv/consumer` module.
+
+:::caution
+The upgrade height in the proposal should correspond to a height that is after the `spawn_time` in the consumer addition proposal submitted to the `provider` chain.
+:::
+
+Otherwise, the upgrade is indistinguishable from a regular on-chain upgrade proposal.
+
+### 3. spawn time is reached
+
+When the `spawn_time` is reached on the `provider` it will generate a `ConsumerGenesis` that contains the validator set that will supercede the `sovereign` validator set.
+
+This `ConsumerGenesis` must be available on the sovereign chain during the on-chain upgrade.
```

Comment:
Should we add in a note about having to manually mutate the JSON to set PreCCV = true:?

Reply:
Good catcH!